### PR TITLE
Added MCP Playwright support

### PR DESCRIPTION
**Pull request by executeautomation**
Added MCP Playwright support

Published in NPM package https://www.npmjs.com/package/@executeautomation/playwright-mcp-server

### CODE REVIEW

**Comment by michaellatman** [comment]
Hello! Thank you. Can you please set runtime to node and put your license? Thanks!

**Comment by michaellatman** [comment]
Also the package name must exactly match the published npm package name, otherwise the installs won't work

File: packages/package-list.json [hunk 1]
```diff
@@ -260,5 +259,11 @@
     "homepage": "https://github.com/mcp-get/community-servers#readme",
     "license": "MIT",
     "runtime": "node"
-  }
+  },
+  {
+    "name": "playwright-mcp-server",
```
Comment (michaellatman) [inline packages/package-list.json:264]: Align package name to the name published to npm

File: packages/package-list.json [hunk 2]
```diff
@@ -260,5 +259,11 @@
...
+    "description": "A Model Context Protocol server for Playwright for Browser Automation and Web Scraping.",
+    "vendor": "ExecuteAutomation, Ltd (https://executeautomation.com)",
+    "sourceUrl": "https://github.com/executeautomation/mcp-playwright/tree/main/src", 
+    "homepage": "https://github.com/executeautomation/mcp-playwright",
```
Comment (michaellatman) [inline packages/package-list.json:268]: Add license and runtime (node)

File: packages/package-list.json (hunk 1 above)
Comment (executeautomation) [reply packages/package-list.json:264]: Ah that should be a merge conflict, sorry about that

File: packages/package-list.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (executeautomation) [reply packages/package-list.json:268]: Bummer, fixed it :)

Comment (executeautomation) [reply packages/package-list.json:268]: Done, thanks !

File: packages/package-list.json (hunk 1 above)
Comment (michaellatman) [reply packages/package-list.json:264]: I think it should be @executeautomation/playwright-mcp-server

Comment (executeautomation) [reply packages/package-list.json]: Ah great, thanks for that, updated it !

**Pull request merged**